src/components/players/PlayerDetails.js

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
-import { connect } from 'react-redux';
 
-const PlayerDetails = (props)=> {
-    const { selectPlayer } = props;
+import { connect } from 'react-redux';
```

Comment:
Remove space. Both react-redux and react are 3rd party files and should be grouped together. Perhaps my slack explanation was misinterpreted. All good.